src/ajax-adapter.js

```diff
@@ -142,6 +146,7 @@ export class AjaxAdapter {
       },
       response => {
         var responseInput = new HttpResponse(response, requestInfo.zConfig);
+        if (!response.json) { responseInput.error = response; requestInfo.error(responseInput); return; }
```

Comment:
thanks @sickboy- can you bring me up to speed on the scenario that requires this line of code.  I think I know but I want to make sure so I can add a test.  Also wondering if `response.text()` should be used instead.